docs/list_of_tutorials.jl

```diff
@@ -64,6 +66,8 @@ if generate_tutorials
     skip_execute = [
         "Atmos/dry_rayleigh_benard.jl",               # takes too long
         "Atmos/risingbubble.jl",                      # broken
+        "Atmos/agnesi_hs_lin.jl",                     # broken
```

Comment:
Sorry for flip-flopping. If these tutorials are not generating valid markdown as indicated, then let's remove them from `skip_execute` and `tutorials` above. The `heat_equation.jl` and `heldsuarez.jl` tutorials generate valid markdown, for reference, if you'd like to make the necessary changes and if you think they would make good tutorials.

In particular, the main issue with not generating valid markdown is that, inside functions, double #'s must be used for comments to tell Literate to not break the function into multiple example blocks (`heldsuarez.jl` uses this extensively).

Reply:
thanks @charleskawczynski let me take a look at how that is done. However, this may take me some time because I am not on this full time. What do you suggest in the meantime so that these drivers are kept up to date with master? These are important tests to always makes sure things are not breaking. 
Thanks